extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -68,6 +79,18 @@ Adds support for Baidu Analytics. More details for adding Baidu Analytics suppor
 
 <!-- markdown-link-check-enable -->
 
+### bg.canary
+
+Type attribute value: `bg.canary`
+
+DO NOT SUBMIT: Add a paragraph to describe bg.canary.
+
+### bg
+
+Type attribute value: `bg`
+
+DO NOT SUBMIT: Add a paragraph to describe bg.
```

Comment:
@micajuine-ho Are these relevant?

Reply:
bg and bg.canary are special cases the iframe transport situation. They were added in this PR https://github.com/ampproject/amphtml/pull/11712 but I don't have any insight to who they are.

Reply:
I'll comment these out. If they face the user somehow we might need to bring them back.